src/pages/job-apply/my-jobs/my-jobs.services.tsx

```diff
@@ -117,6 +120,28 @@ function setStatusJSX(status: keyof typeof APPLICANT_STATUS) {
     alignItems: 'center',
     fontSize: 'var(--font-size-s)',
   };
+  const confirmedStyles: CSSProperties = {
+    padding: '10px',
+    height: '1.5rem',
+    backgroundColor: 'var(--color-success-01)',
+    borderRadius: '25px',
+    display: 'flex',
+    color: '#fff',
+    justifyContent: 'center',
+    alignItems: 'center',
+    fontSize: 'var(--font-size-s)',
+  };
+  const completedStyles: CSSProperties = {
+    padding: '10px',
+    height: '1.5rem',
+    backgroundColor: 'var(--color-gray-04)',
+    borderRadius: '25px',
+    display: 'flex',
+    color: '#333',
```

Comment:
pls avoid using hard-coded colors and use variables(add if it doesn't exist)

Reply:
This was legacy from mohammadali branch. I fixed replaced the hardcoded colors with variables.